DeadCrumbs/app/src/main/java/dead/crumbs/ui/MainActivity.kt

```diff
@@ -70,6 +75,36 @@ class MainActivity : AppCompatActivity() {
         }
     }
 
+    //------------Dead Reckoning-------------//
+
+    private lateinit var drService: DeadReckoningService
+    private var drBound: Boolean = false
```

Comment:
This is something me and Bjarke included in the Bluetooth counterpart but is has become unnecessary/obsolete.

Reply:
Removed it